src/test/java/com/wcc/platform/integrationtests/CmsServiceIntegrationTest.java

```diff
@@ -33,15 +32,24 @@
 class CmsServiceIntegrationTest extends SurrealDbIntegrationTest {
 
   @Autowired private CmsService service;
+  @Autowired private PageRepository pageRepository;
+  @Autowired private ObjectMapper objectMapper;
+
+  @BeforeEach
```

Comment:
Shouldn't it be '@AfterEach' - like teardown after tests finished ?

Reply:
on this case I prefer before each, because I am creating the data in the test, but both ways works in most of the cases.